Model/Source/DefaultOptions.php

```diff
@@ -29,14 +30,41 @@ class DefaultOptions
 {
     // Maximum characters length of company name.
     private const COMPANY_NAME_MAX_LENGTH    = 50;
+    /**
+     * @deprecated
+     */
     private const INSURANCE_BELGIUM          = 'insurance_belgium_custom';
-    private const INSURANCE_BELGIUM_AMOUNT   = 500;
+    /**
+     * @deprecated
+     */
     private const INSURANCE_EU_AMOUNT_50     = 'insurance_eu_50';
+    /**
+     * @deprecated
+     */
     private const INSURANCE_EU_AMOUNT_500    = 'insurance_eu_500';
+    /**
+     * @deprecated
+     */
     private const INSURANCE_AMOUNT_100       = 'insurance_100';
+    /**
+     * @deprecated
+     */
     private const INSURANCE_AMOUNT_250       = 'insurance_250';
+    /**
+     * @deprecated
+     */
     private const INSURANCE_AMOUNT_500       = 'insurance_500';
+    /**
```

Comment:
hoeft niet, is maar een tip mocht je dit niet weten, maar je kunt deze ook in 1 regel schrijven
```php
/** @deprecated */
```

dan neemt het niet zoveel ruimte in beslag :)

Reply:
Yes, bedankt voor de tip. Ik heb het aangepast.